chapters/functions.tex

```diff
@@ -2372,6 +2373,8 @@ \subsection{Annotations for External Functions}\label{annotations-for-external-l
   In case calls to several external functions are generated in the same translation unit, the \lstinline!Include! annotations of the different functions must not define the same function -- except when relying on the deprecated behavior.
 
   The included code should be valid C89 code.
+  If the code uses arrays the tool is responsible for ensuring that a C-header defining \lstinline[language=C]!size_t! is included before this code.
```

Comment:
One shouldn't need to inspect the "code".  When trying to come up with a precise rule, I can't think of anything better than going for the `size`-expression:
```suggestion
  If the \lstinline[language=grammar]!external-function-call! contains any `size`-expression, the tool is responsible for ensuring that a C-header defining \lstinline[language=C]!size_t! is included before the \lstinline!"insertedCode"!.
```